### PR TITLE
Drop support for EL 7.

### PR DESCRIPTION
**Pull request by bowlofeggs**
This is critical priority because it is blocking https://github.com/fedora-infra/bodhi/pull/1697.

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Review by igor-raits — APPROVED** [review]

**Review by mcurlej — APPROVED** [review]
LGFM

File: docs/index.rst [hunk 1]
```diff
@@ -26,6 +26,12 @@ Features
 * Queries ResultsDB for automated test results and displays them on updates.
 
 
+System Requirements
+-------------------
+
+Bodhi is currently supported on all active Fedora releases.
```
Comment (pypingou) [inline docs/index.rst]: + only?

**Comment by pypingou** [comment]
👍 for me

File: docs/index.rst (hunk 1 above)
Comment (bowlofeggs) [reply docs/index.rst]: I changed it to "Bodhi is currently only supported on active Fedora releases."

**Comment by pypingou** [comment]
👍

**Comment by bowlofeggs** [comment]
The test failure is due to there no longer being a nosetests.xml file. Instead there are now nosetests-{release}.xml files in the test_results/ folder. I'm investigating whether Jenkins' JUnit plugin can process multiple test results files in a single job or not.

**Comment by bowlofeggs** [comment]
https://github.com/fedora-infra/bodhi/pull/1856 should fix this PR once it is merged and the CI service job runs.

**Comment by bowlofeggs** [comment]
jenkies test

**Pull request merged**